src/Positionables/Positionable.ts

```diff
@@ -45,7 +45,7 @@ export default class Positionable {
 
         if(this.parent != null) {
             let parentAbsPos = this.parent.absolutePosition;
-            let magnitude = MathUtil.length(this.x, this.y);
+            let magnitude = MathUtil.vectorLength(this);
```

Comment:
While this works because of duck typing, won't this break when methods are properly TypeScript tagged with a type? I'm leaning on JS' soft typing here and it's probably a bad idea. Anything with the correct properties should probably implement a vector interface and the MathUtil methods should expect an interface instance?

I'm not sure about this but, like other comments, it might be worth filing an issue to investigate in case this has consequences.